addons/mass_mailing/static/src/scss/themes/theme_default.scss

```diff
@@ -176,6 +176,11 @@ div.col:not([align]) {
     border-collapse:separate;
 }
 
+// Prevent the website snippets sidebar from overlapping the template
+body.has_snippets_sidebar .o_layout {
+    padding-right: 288px !important; // snippets sidebar width
```

Comment:
Is it possible to use `$o-we-sidebar-width` instead ? This will be more change-proof :)

Reply:
Hello :slightly_smiling_face: I think I already tried that but it couldn't find the variable from this file :confused: I'm going to give it another go and rebase since I'm at it :smile: